### PR TITLE
zib-wounds.WoundImage

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: known-issues.yml [hunk 1]
```diff
@@ -730,6 +730,15 @@ zib-VisualFunction.VisualAid.Product:
     Device.udiCarrier:*UdiCarrier.carrierHRF:
       - datatype: string instead of CD compatible datatype
         reason: The mapping of ProductID is not straightforward and relates to multiple elements in FHIR. In result, the mapping of the concept to FHIR will not match on a datatype level.
+        
+zib-wounds.WoundImage:
+  zib deviations:
+    DocumentReference.content.attachment.url:
+      - alias: WondFoto instead of Wondfoto
+        for: NL-CM:19.4.11 (Burnwound.WoundImage)
```
Comment (LuudSlagter) [inline known-issues.yml:738]: There still is a QA error present regarding this deviation. Should the known issue be formulated somewhat differently?

File: resources/nl-core/nl-core-wounds.WoundImage.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-burnwound-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/Burnwound-v3.4(2020EN)" />
+    <name value="zib Burnwound-v3.4(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-pressureulcer-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN)" />
+    <name value="zib Wound-v3.3(2020EN)" />
```
Comment (LuudSlagter) [inline resources/nl-core/nl-core-wounds.WoundImage.xml:29]: This name is incorrect.

File: resources/zib/zib-wounds.WoundImage.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.WoundImage" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-wounds.WoundImage" />
```
Comment (LuudSlagter) [inline resources/zib/zib-wounds.WoundImage.xml:4]: See comment in WoundCharacteristics.

File: resources/zib/zib-wounds.WoundImage.xml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A photo of the wound, burn wound or decubitus wound as visual information." />
+  <purpose value="This DocumentReference resource represents the WoundImage concept from Dutch zibs ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [Burnwound v3.4(2020EN)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN)), [PressureUlcer v3.3(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN)) and [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))." />
```
Comment (LuudSlagter) [inline resources/zib/zib-wounds.WoundImage.xml:18]: Please align the purpose to the format I used in https://github.com/Nictiz/Nictiz-R4-zib2020/pull/407.

File: resources/zib/zib-wounds.WoundImage.xml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A photo of the wound, burn wound or decubitus wound as visual information." />
```
Comment (LuudSlagter) [inline resources/zib/zib-wounds.WoundImage.xml:17]: I think we should align the descriptions of the WoundCharacteristics and WoundImage profiles.

File: known-issues.yml [hunk 6]
```diff
@@ -730,6 +730,15 @@ zib-VisualFunction.VisualAid.Product:
...
+        reason: The zib concept name "Wondfoto" is incorrect and must be corrected to "WondFoto".
+      - datatype: url instead of ED
+        reason: to be filled.
```
Comment (LuudSlagter) [inline known-issues.yml:741]: Don't forget to update this reason ;)

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: known-issues.yml (hunk 1 above)
Comment (Ashwin-nictiz) [reply known-issues.yml:738]: @pieter-edelman-nictiz will look into it

File: resources/nl-core/nl-core-wounds.WoundImage.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-wounds.WoundImage.xml:29]: Fix

File: resources/zib/zib-wounds.WoundImage.xml (hunk 4 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundImage.xml]: Fix

File: resources/zib/zib-wounds.WoundImage.xml (hunk 5 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundImage.xml:17]: Fix

File: known-issues.yml (hunk 1 above)
Comment (Ashwin-nictiz) [reply known-issues.yml:746]: Fixed by @pieter-edelman-nictiz

File: known-issues.yml (hunk 6 above)
Comment (pieter-edelman-nictiz) [reply known-issues.yml]: In line with the update profiling guidelines, the mapping has been moved and this comment has been removed.

**Pull request closed without merging**